web-console/src/lib/compositions/streaming/inspection/useTableUpdater.ts

```diff
@@ -79,21 +82,24 @@ export function useTableUpdater() {
   const [{ rowsCallback, pause, resume }, setState] = useState(resumed({ rowsCallback: () => new Map() }))
   const readStream = useCallback(
     async (
-      url: URL,
-      requestedNeighborhood: NeighborhoodQuery,
+      egressParams: Arguments<typeof HttpInputOutputService.httpOutput>,
       setRows: Dispatch<SetStateAction<Row[]>>,
       setLoading: Dispatch<SetStateAction<boolean>>,
       relation: Relation,
       controller: AbortController
     ) => {
       // We try and fetch one more row than requested,
       // this helps to detect if this is the last page.
+      // TODO:
```

Comment:
isn't it enough to have this comment once in the other file?

Reply:
I wanted to reeeally make sure this is not missed

Reply:
But when this is fixed, does this place need to be corrected, or just the implementation?
If the latter, only one comment is enough.